tests/app/questionnaire/test_list_action.py

```diff
@@ -0,0 +1,64 @@
+import pytest
```

Comment:
Not sure we were testing handlers?

Reply:
We don't, at least not list action. It's because this resume panel bug cannot be recreated in test schema, as far as I am aware. I mocked a list action situation from ccs schema.

Reply:
I think this test highlights why we don't currently unit test the handlers - so many mocks are needed that it's not clear what is being tested, and there is a high chance we end up testing something that can't happen with schemas. I think we should replace this with a test schema and an integration test.

Reply:
This test has been replaced with integration test, thanks!